catalog/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "tagline": "[30 characters]",
+  "tagline": "Get your entity tagging sorted",
   "repository": "https://github.com/newrelic/nr1-tag-improver.git",
   "details": "[]",
```

Comment:
Is there a "long form" version of the tagline you might be able to add here? This is what will populate the main details on the initial landing page for app info in the catalog.

Reply:
Here's a bit more of a visual breakdown of what the config maps to
![New_Relic_One_catalog___New_Relic_One](https://user-images.githubusercontent.com/12112563/103934907-a79b5680-50da-11eb-8038-29f601e0569c.png)